specification/apicenter/DataApi/models/api.tsp

```diff
@@ -0,0 +1,120 @@
+namespace ApiCenter.DataApi.Models;
+
+@doc("API contract")
+model Api {
+  @doc("The resource name.")
+  @maxLength(50)
+  name: string;
+
+  @doc("The name of the API.")
+  @maxLength(50)
+  title: string;
+
+  @doc("Short descrption of the API.")
+  @maxLength(200)
+  summary?: string;
+
+  @doc("The description of the API")
+  @maxLength(1000)
+  description?: string;
+
+  @doc("Kind of API. For example, REST or GraphQL.")
+  kind: string;
+
+  @doc("Current lifecycle stage of the API.")
+  lifecycleStage?: ApiLifecycleStage;
+
+  @doc("Terms of service for the API.")
+  termsOfService?: ApiTermsOfService;
+
+  @doc("The license information for the API.")
+  license?: ApiLicenseModel;
+
+  @doc("External documentation")
+  externalDocumentation?: Array<ApiExternalDocumentation>;
+
+  @doc("Points of contact for the API.")
+  contacts?: Array<ApiContact>;
+
+  @doc("The custom metadata defined for API entries.")
+  customProperties?: Record<string>;
+
+  @doc("Last updated date and time.")
+  lastUpdated: utcDateTime;
+}
+
+@doc("Api External Documentation Model")
+model ApiExternalDocumentation {
+  @doc("Title of the documentation.")
+  @maxLength(50)
+  title: string;
+
+  @doc("Description of the documentation.")
+  @maxLength(1000)
+  description: string;
+
+  @doc("URL pointing to the documentation.")
+  @maxLength(200)
+  url: string;
```

Comment:
Should be of `url` type: https://microsoft.github.io/typespec/language-basics/built-in-types#string-types

Reply:
As should all URLs throughout.

Reply:
Fixed in latest revision.